src/test/cli/rbd/help.t

```diff
@@ -26,7 +26,7 @@
       config pool set                   Set a pool-level configuration override.
       copy (cp)                         Copy src image to dest.
       create                            Create an empty image.
-      deep copy (deep cp)               Deep copy src image to dest.
+      deep copy (deep cp)               Deep copy (with snapshots) src to dest.
```

Comment:
```suggestion
      deep copy (deep cp)               Deep copy (including snapshots) src image to dest.
```

Reply:
And similarly in those two other places.

Reply:
Well, I wanted to keep the `rbd --help` output from overflowing the standard 80-column terminal line. There are lines which overflow already, but I consider it ugly, and tried to avoid it. If the long form is better, I can rewrite this according to your suggestion.

Reply:
Please do.  Since there already are lines that overflow 80-column limit, let's not drop words from the description just to meet it.